index.js

```diff
@@ -30,7 +30,7 @@ export async function generate(schemaName, { count } = {}) {
   for (let i = 0; i < numberToGenerate; i++) {
     result.push(
       /** @type {import('@mapeo/schema').MapeoDoc} */ (
-        await JSONSchemaFaker.resolve(createFakerSchema(targetSchema))
+        JSONSchemaFaker.generate(createFakerSchema(targetSchema))
```

Comment:
originally used `resolve()` because their usage docs suggested, but not sure if that applies for us:
 https://github.com/json-schema-faker/json-schema-faker/blob/master/docs/USAGE.md#example-usage

also there's apparently a difference in resolving refs between the two:

> jsf.generate() is the sync-version, with partial dereferencing through given refs, etc.
> jsf.resolve() is the async-version, with full dereferencing, given refs are also supported.

my guess is that resolve will work if you have refs that point to urls, which doesn't apply to us.

anyways, just wanted to check but if refs are being resolved as expected (usually would check the location property of an observation), think this change makes sense 👍

Reply:
Yeah my understanding is that for the sync version you need to pass the refs as the second parameter, but with the async version it will try to resolve them either from disk or url. It was working ok for me with generate. Have to say it's a very unusual API!